.changeset/curly-windows-learn.md

```diff
@@ -0,0 +1,11 @@
+---
+'@xstate/graph': major
+---
+
+@author: @davidkpiano
+
+Changed `getSimplePaths` to `getSimplePlans`, and `getShortestPaths` to `getShortestPlans`. Both of these functions now return `StatePlan[]`.
+
+Added functions `traverseSimplePlans`, `traverseSimplePlansTo` and `traverseSimplePlansFromTo`, to allow for traversal of `StatePlan[]` generated by `getSimplePlans`.
+
+Added functions `traverseShortestPaths` and `traverseShortestPlansFromTo` to allow for traversal of `StatePlan[]` generated by `getShortestPlans`.
```

Comment:
Those two are not correct - `traverse*` are used within respective `get*`. So `traverse*` return exactly the same thing as `get*`, the difference is what kind of arguments they receive:
- `get*` - machine as argument
- `traverse*` - behavior as argument

Reply:
👍